content/en/docs/getting-started/_index.md

```diff
@@ -1,18 +1,21 @@
 ---
 title: Getting Started
 description: Getting started with Falco
-weight: 20
+weight: 10
 ---
-You can deploy Falco on a local machine, cloud, a managed Kubernetes cluster, or a Kubernetes cluster such as K3s running on IoT & Edge computing.
+
+## What is Falco
+ 
+ Falco, a CNCF incubating project, provides near real-time threat detection for cloud, container, and Kubernetes workloads by leveraging runtime insights. Falco is able to monitor events from a variety of sources including the Linux kernel, Kubernete API server, container runtime, and more. 
```

Comment:
```suggestion
Falco is a cloud-native security tool. It provides near real-time threat detection for cloud, container, and Kubernetes workloads by leveraging runtime insights. Falco can monitor events from various sources, including the Linux kernel, and enrich them with metadata from the Kubernetes API server, container runtime, and more.
```

Reply:
@nate-double-u I've adapted it to match the latest description on the main page. Let me know if it's ok! :pray: